var/spack/repos/builtin/packages/c-blosc2/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class CBlosc2(CMakePackage):
+    """Next generation c-blosc with a new API, a new container and
+       other bells and whistles"""
+
+    homepage = "http://www.blosc.org"
+    url      = "https://github.com/Blosc/c-blosc2/archive/v2.0.0-beta.1.tar.gz"
+    git      = "https://github.com/Blosc/c-blosc2.git"
+
+    maintainers = ['ax3l']
+
+    version('develop', branch='master')
```

Comment:
I would wait for the `2.0.0` release (currently: `2.0.0-beta1`) before tagging numbered versions.
By that, we can keep the package simple and clean in case things are still changing and potential install bugs are fixed.